sympy/stats/joint_rv_types.py

```diff
@@ -1,24 +1,27 @@
-from sympy import (sympify, S, pi, sqrt, exp, Lambda, Indexed, Gt, IndexedBase,
-                    besselk, gamma, Interval, Range, factorial, Mul, Integer,
-                    Add, rf, Eq, Piecewise, ones, Symbol, Pow, Rational, Sum,
-                  imageset, Intersection, Matrix, symbols, Product, IndexedBase)
+from sympy import (sympify, S, pi, sqrt, exp, Lambda, Indexed,
+                   besselk, gamma, Interval, Range, factorial, Mul, Integer,
+                   Add, rf, Eq, Piecewise, ones, Symbol, Pow, Rational, Sum,
+                   Intersection, Matrix, symbols, Product, IndexedBase)
 from sympy.matrices import ImmutableMatrix
 from sympy.matrices.expressions.determinant import det
 from sympy.stats.joint_rv import (JointDistribution, JointPSpace,
-    JointDistributionHandmade, MarginalDistribution)
+                                  JointDistributionHandmade, MarginalDistribution)
 from sympy.stats.rv import _value_check, random_symbols
 
 __all__ = ['JointRV',
-'Dirichlet',
-'GeneralizedMultivariateLogGamma',
-'GeneralizedMultivariateLogGammaOmega',
-'Multinomial',
-'MultivariateBeta',
-'MultivariateEwens',
-'MultivariateT',
-'NegativeMultinomial',
-'NormalGamma'
-]
+           'Dirichlet',
```

Comment:
I think this indentation wasn't required. In other modules, AFAIK, there isn't any.

Reply:
I can remove this.